fix for version 0.46 (#1365)
facebook/react-native-website: website/versioned_docs/version-0.46/animated.md

original_id: animated
---

> The `Animated` library is designed to make animations fluid, powerful, and to be built and maintained in brief. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and basic `start`/`stop` methods to control time-based animation execution.

Reviewer: ```suggestion
The `Animated` library is designed to make animations fluid, powerful, and painless to build and maintain. `Animated` focuses on declarative relationships between inputs and outputs, with configurable transforms in between, and `start`/`stop` methods to control time-based animation execution.
```

### Animatable components

> Only animatable components can be animated. These reserved components do the magic of binding the animated values to the properties, and do targeted native updates to avoid the cost of the react render and reconciliation process on every frame. They also handle cleanup on unmount so they are safe by default.

Reviewer: ```suggestion
Only animatable components can be animated. These components do the magic of binding the animated values to the properties, and do targeted native updates to avoid the cost of the react render and reconciliation process on every frame. They also handle cleanup on unmount so they are safe by default.
```

- [`Animated.sequence()`](animated.md#sequence) starts the animations in order, waiting for each to complete before starting the next.
- [`Animated.stagger()`](animated.md#stagger) starts animations in order and in parallel, but with successive delays.

> Animations can also be chained together in brief by setting the `toValue` of one animation to be another `Animated.Value`. See [Tracking dynamic values](animations.md#tracking-dynamic-values) in the Animations guide.

Reviewer: ```suggestion
Animations can also be chained together by setting the `toValue` of one animation to be another `Animated.Value`. See [Tracking dynamic values](animations.md#tracking-dynamic-values) in the Animations guide.
```
